test/lib/jdk/test/lib/security/SecurityUtils.java

```diff
@@ -70,5 +70,32 @@ private static void removeFromDisabledAlgs(String prop, List<String> algs) {
         Security.setProperty(prop, value);
     }
 
+    /**
+     * Removes the specified algorithms from the
+     * jdk.xml.dsig.secureValidationPolicy security property. Matches any
+     * part of the algorithm URI.
+     */
+    public static void removeAlgsFromDSigPolicy(List<String> algs) {
```

Comment:
How about use `String... algs` as arguments?

Reply:
Yes, that is nicer.